robottelo/cli/docker.py

```diff
@@ -233,37 +233,37 @@ class DockerImage(Base):
 
     Subcommands::
 
-        info                          Show a docker image
-        list                          List docker_images
+        info                          Show a docker manifest
+        list                          List docker_manifests
 
     """
-    command_base = 'docker image'
+    command_base = 'docker manifest'
 
     @classmethod
     def info(cls, options=None):
-        """Gets information about docker images
+        """Gets information about docker manifests
 
         Usage::
 
-            hammer docker image info [OPTIONS]
+            hammer docker manifest info [OPTIONS]
 
         Options::
 
-            --id ID                       a docker image identifier
+            --id ID                       a docker manifest identifier
             --name NAME                   Name to search by
             --repository REPOSITORY_NAME  Repository name to search by
             --repository-id REPOSITORY_ID repository ID
 
         """
-        return super(DockerImage, cls).info(options)
+        return super(DockerManifest, cls).info(options)
 
     @classmethod
     def list(cls, options=None, per_page=True):
-        """List docker images
+        """List docker manifests
 
         Usage::
 
-            hammer docker image list [OPTIONS]
+            hammer docker manifest list [OPTIONS]
```

Comment:
I can see that `hammer docker manifest list` command has more option. Please, update options list if you like